File: mobile/src/main/java/rs/readahead/washington/mobile/views/interfaces/IGalleryMediaHandler.java
package rs.readahead.washington.mobile.views.interfaces;

import com.hzontal.tella_vault.VaultFile;



public interface IGalleryMediaHandler {
    void playMedia(VaultFile mediaFile);
    void onSelectionNumChange(int num);
    void onMediaSelected(VaultFile mediaFile);
    void onMediaDeselected(VaultFile mediaFile);
}

File: mobile/src/main/java/rs/readahead/washington/mobile/views/fragment/vault/adapters/attachments/IGalleryMediaHandler.kt
package rs.readahead.washington.mobile.views.fragment.vault.adapters.attachments

import rs.readahead.washington.mobile.views.fragment.vault.adapters.viewholders.data.VaultFile

interface IGalleryMediaHandler {
    fun playMedia(vaultFile: VaultFile)
    fun onSelectionNumChange(num: Int)
    fun onMediaSelected(vaultFile: VaultFile)
    fun onMediaDeselected(vaultFile: VaultFile)
}